{"uuid":"76e100da-282a-4310-b976-7d8ab4aca470","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["21368f7a-a50e-4ed2-a923-187066c4dfa4"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561105705,"stop":1696561105705},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561105884,"stop":1696561105884},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561105995,"stop":1696561105995},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561108051,"stop":1696561108052}],"start":1696561105129,"stop":1696561108052}